pino-elasticsearch.js

```diff
@@ -112,7 +109,10 @@ if (require.main === module) {
       version: 'v',
       help: 'h',
       host: 'H',
-      port: 'p'
+      port: 'p',
+      index: 'i',
+      size: 's',
+      loglevel: 'l'
```

Comment:
can you please change this option long name to something es-specific. Pino is a tool for a logger, but this tool can log as well. It might be confusing.

Also, you should add those options to `usage.txt`.